packages/ckeditor5-dev-env/lib/release-tools/utils/executeondependencies.js

```diff
@@ -26,7 +28,7 @@ const getPackageJson = require( './getpackagejson' );
  */
 module.exports = function executeOnDependencies( options, functionToExecute ) {
 	const packagesAbsolutePath = path.join( options.cwd, options.packages );
-	const directories = workspaceUtils.getDirectories( packagesAbsolutePath );
+	const directories = tools.getDirectories( packagesAbsolutePath );
```

Comment:
Because we check whether the directory is a valid package, we can take everything from the CWD (instead of directories starting with `ckeditor5-`).